walmart/walmart.py

```diff
@@ -49,45 +79,40 @@ def orders(self):
     def report(self):
         return Report(connection=self)
 
-    def get_sign(self, url, method, timestamp):
-        return self.sign_data(
-            '\n'.join([self.consumer_id, url, method, timestamp]) + '\n'
-        )
-
-    def sign_data(self, data):
-        rsakey = RSA.importKey(base64.b64decode(self.private_key))
-        signer = PKCS1_v1_5.new(rsakey)
-        digest = SHA256.new()
-        digest.update(data.encode('utf-8'))
-        sign = signer.sign(digest)
-        return base64.b64encode(sign)
-
-    def get_headers(self, url, method):
-        timestamp = str(int(round(time.time() * 1000)))
-        return {
-            'WM_SEC.AUTH_SIGNATURE': self.get_sign(url, method, timestamp),
-            'WM_SEC.TIMESTAMP': timestamp,
-            'WM_QOS.CORRELATION_ID': str(uuid4()),
-        }
-
     def send_request(
         self, method, url, params=None, body=None, request_headers=None
     ):
-        encoded_url = url
-        if params:
-            encoded_url += '?%s' % urlencode(params)
-        headers = self.get_headers(encoded_url, method)
+        headers = {
```

Comment:
can you explain this and what this is?

Reply:
`A unique ID which identifies each API call and used to track and debug issues; use a random generated GUID for this ID`